xml/gl.xml

```diff
@@ -134,11 +134,12 @@ typedef unsigned int GLhandleARB;
         </group>
 
         <group name="ProgramStringProperty">
-            <enum name="GL_PROGRAM_STRING"/>
+            <!-- Note to reviewer: could be PROGRAM_STRING_NV -->
```

Comment:
Let me know what you think about this. I'll remove this comment once approved, but I wasn't sure whether to use GL_PROGRAM_STRING_ARB or GL_PROGRAM_STRING_NV (or both?), so I opted for ARB instead.

```suggestion
```

Reply:
That's weird. I thought I made a comment, but maybe I hit the wrong button. I would go for _ARB.

Reply:
Yeah I thought that'd be the best thing to do. Thanks +1